test/api_test.js

```diff
@@ -58,7 +58,7 @@ describe("Api", () => {
     describe("full URL", () => {
       var endpoints;
 
-      beforeEach(() => endpoints = api.endpoints({fullUrl: true}))
+      beforeEach(() => endpoints = api.endpoints({ fullUrl: true }))
```

Comment:
I prefer ``{data: "toto"}`` rather than ``{ data: "toto" }`` but it is a matter of taste I guess.

@n1k0 what do you think?